mono/mini/debugger-agent.c

```diff
@@ -2714,6 +2692,21 @@ process_suspend (DebuggerTlsData *tls, MonoContext *ctx)
 	suspend_current ();
 }
 
+
+/* Conditionally call process_suspend depending oh the current state */
+static gboolean
+try_process_suspend (DebuggerTlsData *tls, MonoContext *ctx)
+{
+	if (suspend_count > 0) {
+		/* Fastpath during invokes, see in process_suspend () */
+		if (suspend_count - tls->resume_count == 0)
```

Comment:
How about a == b instead of a - b == 0?
Do you need to check if one is greater than the other, or it works in either direction?

Reply:
This was just code motion, I wanted to make it as identical as possible to facilitate review.